test/services/user-group/index.test.js

```diff
@@ -84,6 +84,62 @@ describe('classes service', () => {
 			expect(data[0]._id.toString()).to.equal(classes[0]._id.toString());
 		});
 
+		it('should allow students to only find classes they participate in', async () => {
+			const admin = await testObjects.createTestUser({ roles: ['administrator'] });
```

Comment:
however the test does not match the description, and create class user relation and add them to a testschool